_layouts/assignment.html

```diff
@@ -35,3 +35,45 @@ <h2>Submission Instructions</h2>
 	{% capture instr %}{% include_relative submission_instructions.md %}{% endcapture %}
 	{{ instr | markdownify }}
 {% endunless %}
+
+{% if page.rubric %}
+	<hr>
+	<table>
+	{%- assign total = 0 %}
+	<thead><tr>
+		<th>Name</th>
+		<th>Points</th>
+		<th>Description</th>
+	</tr></thead>
+	
+	<tbody>
+	{% for item in page.rubric %}
+	<tr>
+		<td>
+			{{ item.name }}
+		</td>
+		<td>
+			{{ item.points }}
+			{% assign temp = item.points | plus: 0 %} <!-- int(item.points) -->
```

Comment:
unnecessary, since `total` starts out as an int. I tested it, it works. Change incoming.